include/psa/crypto_values.h

```diff
@@ -1555,9 +1563,43 @@
  * This lifetime value is the default storage area for the calling
  * application. Implementations may offer other storage areas designated
  * by other lifetime values as implementation-specific extensions.
+ * See ::psa_key_lifetime_location_t for more information.
  */
 #define PSA_KEY_LIFETIME_PERSISTENT             ((psa_key_lifetime_t)0x00000001)
 
+#define PSA_KEY_LIFETIME_PERSISTENCE_VOLATILE ((psa_key_lifetime_persistence_t)0x00)
+#define PSA_KEY_LIFETIME_PERSISTENCE_PRIMARY ((psa_key_lifetime_persistence_t)0x01)
```

Comment:
We have a location with the same value that we describe as the "primary secure element". This made me confused at first about the purpose of this constant.

```suggestion
#define PSA_KEY_LIFETIME_PERSISTENCE_PERSISTENT ((psa_key_lifetime_persistence_t)0x01)
```

Reply:
Yeah, I don't like `PERSISTENCE_PRIMARY` either. But `PERSISTENCE_PERSISTENT` is weird and misleading: other `PERSISTENT_xxx` values are also persistent. And I don't like `PERSISTENCE_DEFAULT` either (for example, when you create a key, the persistence defaults to `VOLATILE`, not to `DEFAULT`). Does anyone have a better suggestion?

Reply:
Well it might be a little bit overkill, but how about `PSA_KEY_LIFETIME_PERSISTENCE_PERSISTENT_DEFAULT`?

Reply:
Another candidate perhaps: `PSA_KEY_LIFETIME_PERSISTENCE_STANDARD`?

Reply:
Standard works for me

Reply:
I don't like “standard”: which standard? I [went with “default”](https://github.com/ARMmbed/mbedtls/pull/3302/commits/87977450e154398e5f0b0422d2d93515dc70973e).